testsuite/shared/src/main/java/org/jboss/as/test/integration/management/extension/customcontext/testbase/CustomManagementContextTestBase.java

```diff
@@ -141,14 +142,27 @@ private void test(final ManagementClient managementClient) throws IOException {
         final String staticUrlDirectory = urlBase + "static/";
         final String badStaticUrl = urlBase + "static/bad.txt";
         final String errorUrl = urlBase + "error/index.html";
+        final String securedDynamicUrl = urlBase + "secured-dynamic/";
+        final String unsecuredDynamicUrl = urlBase + "unsecured-dynamic/";
 
```

Comment:
I tried without trailing `/` and tests are passing too. Probably not worth specific test as this use-case will be covered by `/health` endpoint in mp health work